sdk/python/Makefile

```diff
@@ -0,0 +1,6 @@
+
+build:
+	./build.sh
+
+upload-to-s3:
```

Comment:
This file probably should not be a part of this PR.

Reply:
removed